lib/travis/build/script/r.rb

```diff
@@ -258,6 +258,10 @@ def script
           # information.
           dump_log("out")
 
+          # If a fold (like dump log) is the last command it is not folded, so
+          # add a dummy command
+          sh.cmd 'TRUE', echo: false
```

Comment:
I don't think this works. If you want a command that does nothing, you can try `true`, `;`, or `:`.